plugins/formal-verification/build.gradle.kts

```diff
@@ -5,7 +5,16 @@ plugins {
     id("jps-compatible")
 }
 
+repositories {
+    mavenLocal()
+}
+
+
 dependencies {
+
+    implementation("viper:silver_2.13:0.1-SNAPSHOT")
```

Comment:
Do we run the risk of the Viper folks updating to a new version and these instructions going out of date?

Reply:
The version name is identified by the scala's compiler version, it doesn't seem that it is updated often!